src/Shape/Whisker.js

```diff
@@ -110,22 +57,68 @@ export default class Whisker extends BaseClass {
         .style("height", `${window.innerHeight}px`)
         .style("display", "block").node());
     }
-    
-    this._whiskerData(this._data);
 
-    this._shapes.push(new Line()
-      .data(this._lineData)
+    const lineData = [];
+    this._data.forEach((d, i) => {
+
+      const orient = this._orient(d, i);
+
+      let endpointX = this._x(d, i);
+      if (orient === "left") endpointX -= this._length(d, i);
+      else if (orient === "right") endpointX += this._length(d, i);
+
+      let endpointY = this._y(d, i);
+      if (orient === "top") endpointY -= this._length(d, i);
+      else if (orient === "bottom") endpointY += this._length(d, i);
+
+      lineData.push({__d3plus__: true, data: d, i, id: i, x: this._x(d, i), y: this._y(d, i)});
```

Comment:
You are invoking the `this._x` and `this._y` accessors twice for each data point: once here and once to calculate the `endpointX` and `endpointY`. Please calculate them once (ie. `const x = this._x(d, i);`) and then use that new `x` variable in the endpoint calculations.